pkg/controller/backup_controller.go

```diff
@@ -428,6 +428,8 @@ func recordBackupMetrics(backup *api.Backup, backupFile *os.File, serverMetrics
 	backupDuration := backup.Status.CompletionTimestamp.Time.Sub(backup.Status.StartTimestamp.Time)
 	backupDurationSeconds := float64(backupDuration / time.Second)
 	serverMetrics.RegisterBackupDuration(backupScheduleName, backupDurationSeconds)
+	serverMetrics.RegisterVolumeSnapshotsCompletedPerBackup(backupScheduleName)
```

Comment:
We need to include the number of completed snapshots, don't we?

Reply:
I was following what's done for other counter variables: https://github.com/heptio/ark/blob/master/pkg/metrics/metrics.go#L186-L190

Reply:
Do we need to explicitly pass it?

Reply:
Yes. Looking at what @skriss wrote in the issue:
> These two new fields capture the total number of volume snapshots tried, and succeeded, per backup. We should add prometheus metrics to capture these (or the converse - failures).

The way it's coded in this PR now, these 2 new counters are identical to and effectively duplicates of `backupAttemptCount`.

Reply:
Ah, gotcha! Fixing it

Reply:
So here we essentially get that value and set the counter to it, right?